pages.ja/common/tar.md

```diff
@@ -0,0 +1,37 @@
+# tar
+
+> アーカイブ(複数のファイルやフォルダを 1 つのファイルに纏める)為のユーティリティー。
+> gzip や bzip2 などの圧縮方法と組み合わせることが多いです。
+> 詳しくはこちら: <https://www.gnu.org/software/tar>.
+
+- アーカイブを作成し、それにファイルを書き込む:
+
+`tar cf {{出力ファイル名.tar}} {{ファイル1}} {{ファイル2}} {{ファイル3}}`
+
+- gzip 形式で圧縮されたアーカイブを作成し、それにファイルを書き込む:
```

Comment:
Why is `gzip` in there? It isn't in the english version.

Reply:
I checked it out.

The original file did not say 'gzip', but it existed as a passive system, as in 'g[z]ipped'. I translated it.